version.go

```diff
@@ -35,6 +36,35 @@ func (version *ArtifactVersion) refresh() {
 	}
 }
 
+func parseVersionParts(version string) (major uint64, minor uint64, patch uint64, err error) {
```

Comment:
https://github.com/mongodb/curator/blob/master/versions.go should be all that we need for parsing mongodb versions... maybe it makes sense to move that file (and its tests) here...

Reply:
Imported from curator.